### DISCUSSION TITLE
Can I also develop ESP32 BLE or Bluetooth?

### DISCUSSION THREAD

**Discussion by hmmhmmhm** [microsoft/devicescript#481] [Q&A] [answered]
Is it possible to configure a GATT server with DeviceScript? I am currently configuring a BLE GATT server in Javascript with Espruino and would like to migrate to this project if it becomes available.

**Comment by pelikhan** [comment, marked as answer]
Currently, we do not support the BLE stack on ESP32 and it is unlikely that we will be able to get to it soon. If you have experience with ESP-IDF, we would definitely take a contribution in that space.
